packages/terra-site/src/examples/popup/PopupTopCenter.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Popup from 'terra-popup';
+import Button from 'terra-button';
+import ExamplePopupContent from './ExamplePopupContent';
+
+class PopupTopCenter extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleButtonClick = this.handleButtonClick.bind(this);
+    this.handleRequestClose = this.handleRequestClose.bind(this);
+    this.handleOnChange = this.handleOnChange.bind(this);
+    this.setButtonNode = this.setButtonNode.bind(this);
+    this.getButtonNode = this.getButtonNode.bind(this);
+    this.state = { open: false };
+  }
+
+  setButtonNode(node) {
+    this.buttonNode = node;
+  }
+
+  getButtonNode() {
+    return this.buttonNode;
+  }
+
+  handleButtonClick() {
+    this.setState({ open: true });
+  }
+
+  handleRequestClose() {
+    this.setState({ open: false });
+  }
+
+  handleOnChange() {
+    this.setState({ open: false });
+  }
+
+  render() {
+    return (
+      <div style={{ display: 'inline-block' }} ref={this.setButtonNode}>
```

Comment:
That's the topic of the findDomNode, refCallBack, and wrapping div that was brought up. Once we settle on a strategy I can update this pattern